From 4b8eb26ed5a930b0514f7faabd8bf638ef657741 Mon Sep 17 00:00:00 2001
From: Alex Wang <alex.ts.wang@fii-foxconn.com>
Date: Thu, 17 Jul 2025 05:58:12 +0800
Subject: [PATCH 1/1] meta-facebook: clemente: configurations: Fix bus index
 offset

Adjusted MUX channel-to-bus mapping to ensure accurate access to the
CX8 temperature sensor.

Change-Id: Iad8a631d4a4fa5f4db5fa8bb7958a93062cac7e1
Signed-off-by: Alex Wang <alex.ts.wang@fii-foxconn.com>
---
 configurations/nvidia_gb300_io_board.json | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configurations/nvidia_gb300_io_board.json b/configurations/nvidia_gb300_io_board.json
index 99bc2b3..0f2b997 100644
--- a/configurations/nvidia_gb300_io_board.json
+++ b/configurations/nvidia_gb300_io_board.json
@@ -8,7 +8,7 @@
         },
         {
             "Address": "0x1f",
-            "Bus": "$bus - 5",
+            "Bus": "$bus + 3",
             "Labels": [
                 "temp2"
             ],
@@ -42,7 +42,7 @@
         },
         {
             "Address": "0x1f",
-            "Bus": "$bus - 3",
+            "Bus": "$bus + 6",
             "Labels": [
                 "temp2"
             ],
-- 
2.43.0

